app/models/course.rb

```diff
@@ -48,6 +48,11 @@ class Course < ApplicationRecord
     not_required: 9,
   }
 
+  enum :campaign_name,
```

Comment:
Yup i think we should start standardising enums in one place, eg define an initializer in `config/initializers/course_enums.rb` then something like:

```ruby
  COURSE_CAMPAIGN_ENUMS = {
    engineers_teach_physics: "engineers_teach_physics",
  }.freeze

COURSE_CAMPAIGNS = {
  COURSE_CAMPAIGN_ENUMS[:engineers_teach_physics] => 0,
}
```

then in `course.rb`:

```ruby
  enum :campaign_name, COURSE_CAMPAIGNS
```

it'll give us flexibility to easily reference and update the values (if we ever need to), eg `COURSE_CAMPAIGN_ENUMS[:engineers_teach_physics]`